.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -76,19 +76,19 @@ body:
     id: main-memory
     attributes:
       label: How much RAM is available to DMS explicitly?
-      description: Please provide the amount of memory that DMS can really use. This includes main memory (RAM), but also swap or zram. If you're using Kubernetes, provide the resource limit here if you set one.
+      description: Please provide the amount of memory that DMS can really use. This includes physical memory (RAM), but also virtual memory (like swap or zram).  Think about all possible resource restrictions on memory here.
       options:
-        - <= 1GB
-        - <= 2GB
-        - <= 4GB
-        - \> 4GB
+        - <= 1GB (very low)
+        - <= 2GB (low)
+        - <= 4GB (sufficient)
+        - \> 4GB (more than enough)
     validations:
       required: true
   - type: dropdown
     id: cpu-count
     attributes:
-      label: How many CPU cores / threads are available?
-      description: How many CPU cores or threads can DMS use? If you're using Kubernetes, provide the resource limit here if you set one.
+      label: How many CPU cores are available?
+      description: How many CPU cores can DMS use? Think about all possible resource restrictions on CPU cores here.
```

Comment:
> Think about all possible resource restrictions on CPU cores here.

Likewise "Consider any resource limits you may have set that would affect this."

Reply:
Same as above :)